<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubeFeed</class>
 <widget class="QDialog" name="YoutubeFeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Feed</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Feed type:</string>
     </property>
     <property name="buddy">
      <cstring>typeCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="typeCombo">
     <item>
      <property name="text">
       <string>Playlist</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Favorites</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uploads</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>&amp;Playlist ID:</string>
     </property>
     <property name="buddy">
      <cstring>idEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="idEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Start index:</string>
     </property>
     <property name="buddy">
      <cstring>startIndexSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="startIndexSpin">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Max. results:</string>
     </property>
     <property name="buddy">
      <cstring>maxResultsSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="maxResultsSpin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>typeCombo</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>startIndexSpin</tabstop>
  <tabstop>maxResultsSpin</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>YoutubeFeed</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>YoutubeFeed</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>YoutubeFeed</receiver>
   <slot>onTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTypeChanged(int)</slot>
 </slots>
</ui>
